modules/post/osx/capture/keylog_recorder.rb

```diff
@@ -200,7 +165,7 @@ def method_missing(meth, *args, &block)
         server = TCPServer.new(options[:port])
         client = server.accept
         log_semaphore.synchronize do
-          client.puts(log+"\n\r")
+          client.puts(log+"\n")
```

Comment:
Leaving the "\r" in causes the program to throw the following warning:
```
warning: encountered \\r in middle of line, treated as a mere space
```

This error causes the `child_pid` to be incorrectly parsed when it gets output. I'm not a ruby dev, so if there is a suggestion for a workaround here I'm totally open to it.